### PR TITLE
Email alert: send if membership fee not paid after app first approved

### PR DESCRIPTION
**Pull request by weedySeaDragon**
#### PT URL: https://www.pivotaltracker.com/story/show/169956889

_Still need text for the email_ 

## Changes proposed in this pull request:
1.  created requirement for testing if the alert should be sent:
    1. Application is approved **AND**
    2.  EITHER
        - no membership fee payments have ever been made, OR
        - if a membership fee payment has been made in the past, those  (the term) expired before this application was approved.  (Note that once we implement 'users can have more than 1 SHF application,' this will be important.)
    
2. created mail that will be sent (see screenshots below)

3. set the alert schedule and put it into the `load_conditions` task
   - schedule: std_reminder_after_schedule: [2, 9, 14, 30, 60] days after the application was approved

## Screenshots (Optional):

email-2-pay-first-membership-fee-sv
---

email-2-pay-first-membership-fee-en

---
## Ready for review:
@thesuss @AgileVentures/shf-project-team

### CODE REVIEW

**Comment by thesuss** [comment]
I dont see the use for more than one application, I'd rather have the states more "fluid" ... But I'm sure there are good reasons :)

File: config/locales/sv.yml [hunk 1]
```diff
@@ -1252,7 +1252,14 @@ sv:
         message_text:
           expire_alert_html: "Ditt medlemsskap gick ut <span class='date-warning'> %{expire_date}.</span>"
           renew_membership: Du kan förnya ditt medlemsskap och fortsätta dra nytta av medlemsfördelarna genom att logga in och betala.
```
Comment (thesuss) [inline config/locales/sv.yml]: medlemsskap  = medlemskap

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1252,7 +1252,14 @@ sv:
...
-          pay_on_acct_page: "Du betalar via "
+          pay_on_acct_page: &pay_on_acct_page "Du betalar via "
+
+      first_membership_fee_owed:
+        subject: Please pay your SHF Membership fee
+        greeting: *email_greeting
+        message_text:
+          fee_is_unpaid: "Please pay your SHF membership fee...."
```
Comment (thesuss) [inline config/locales/sv.yml]: message_text:
          fee_is_unpaid: Du kan betala ditt medlemskap och börja dra nytta av medlemsfördelarna genom att logga in och betala.

File: config/locales/sv.yml (hunk 1 above)
Comment (weedySeaDragon) [reply config/locales/sv.yml]: oops - I didn't update this one.   <sigh>  I'll do one more commit to fix it.

Comment (thesuss) [reply config/locales/sv.yml]: It can be a separate PR, its a systemic disease

**Pull request merged**